src/server/handlers/content.ts

```diff
@@ -48,3 +53,46 @@ export async function show(req) {
     }
 
 }
+
+export async function create(request, hapi) {
+
+  // parse content transaction
+  // look up on by txid blockchain
+  // if not on chain, broadcast then import
+
+  try {
+
+    const { transactions } = request.payload
+    
+    let records;
+
+    transactions.forEach(async tx => {
```

Comment:
use something compatible with async such as `for (transaction of transactions)` or `Promise.all(transactions.map(async () =>...)`